packages/arb-token-bridge-ui/src/components/common/atoms/Loader.tsx

```diff
@@ -0,0 +1,40 @@
+/*
+    Standardised loading spinner atom
+*/
+
+import Spinner, { LoaderProps as SpinnerProps } from 'react-loader-spinner'
```

Comment:
let's update to 5.3.4
https://www.npmjs.com/package/react-loader-spinner
we're still on 4.0.0

Reply:
Done - changed the API quite a bit. Handled that as well.